packages/e2e/src/specs/contacts/contacts-in-app-navigation.e2e.ts

```diff
@@ -52,22 +52,22 @@ describe("Contacts screen check", () => {
   })
 
   it("Should check if 'phone number' input field is displayed", async () => {
-    const phoneNumber = await ContactsPage.phoneNumberInput
+    const phoneNumber = await ContactsPage.primaryPhoneNumberInput
```

Comment:
That's interesting approach to export new instance of object and every time u import you've got fresh instance. Hope I understand it correctly, but u are using it as class not instance of class, cause it's name starts from capital letter. I would recommend to change it, it might be misleading.